src/components/UserProfile/VolunteeringTimeTab/VolunteeringTimeTab.jsx

```diff
@@ -7,6 +7,9 @@ import { ENDPOINTS } from 'utils/URL';
 import axios from 'axios';
 import styles from './VolunteeringTimeTab.css';
 
+const MINIMUM_WEEK_HOURS = 0;
```

Comment:
I feel like this min_hour should start at 1 since it is the hour of volunteer time.

Reply:
Great point! I thought about this as well but I didn't know if there might be changes later on like a minimum of 5  or 10 being required so I just focused on not allowing negative numbers and setting up a constant that we can update later down the line when we're ready.

Reply:
i think it should be 10 as the currently minimum is 10